docs/api_docs/bundle.yaml

```diff
@@ -764,7 +764,6 @@ definitions:
         type: integer
         format: int64
         minimum: 1
-        readOnly: true
```

Comment:
here `id` is required, setting it as `readOnly` will result in a warning in validation